components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/main/java/org/wso2/carbon/apimgt/rest/api/core/impl/ApisApiServiceImpl.java

```diff
@@ -58,4 +63,39 @@ public Response apisApiIdGatewayConfigGet(String apiId, String accept,
                     MediaType.APPLICATION_JSON).entity(errorDTO).build();
         }
     }
+
+    /**
+     *
```

Comment:
Thank you for pointing out the mistake. This is fixed by https://github.com/wso2/carbon-apimgt/pull/4233/commits/751208cacf54faeaa4f6e01ab3e74456faecd4c8 and https://github.com/wso2/carbon-apimgt/pull/4234/commits/9212f98d36ad3405d06db53edf79880e04108348